product_secondary_unit/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Product Secondary Unit",
+    "summary": "Set a secondary unit per product",
+    "version": "13.0.1.0.0",
+    "development_status": "Beta",
```

Comment:
This module could be in "Production/Stable", If you do this change you also must be add a mantainers key...